src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -337,9 +337,11 @@ void collectTypedefs() {
       if (type == null
```

Comment:
Better first line for this commit: "Minor syntax fixes".  ("Minor fixes" is too vague, hard to know if it's fixing minor clutz bugs or some other thing.)

Reply:
Done.